src/server/server.go

```diff
@@ -2,13 +2,19 @@ package server
 
 import (
 	"fmt"
-	"github.com/gofiber/fiber/v2"
-	"github.com/gofiber/swagger"
-	"gorm.io/gorm"
 	"onepixel_backend/src/config"
 	"onepixel_backend/src/docs"
 	_ "onepixel_backend/src/docs"
 	"onepixel_backend/src/routes/api"
+	"onepixel_backend/src/server/logger"
+
+	"github.com/gofiber/fiber/v2"
+	"github.com/gofiber/swagger"
+	"gorm.io/gorm"
+)
+
+const (
+	ENVIRONMENT_NAME_PRODUCTION = "production"
```

Comment:
I think in Go the culture is not to use ALL_CAP_CASE for variables like Java world no?

Reply:
Constant should use all capital letters and use underscore `_` to separate words. Eg. `INT_MAX`

Reply:
other reference https://www.cnblogs.com/zhangzhihui/articles/17881312.html